thrift/src/main/thrift/atoms/recipe.thrift

```diff
@@ -9,7 +9,6 @@ struct Tags {
   1: required list<string> cuisine
   2: required list<string> category
   3: required list<string> celebration
-  4: required list<string> dietary
```

Comment:
Typically we can't remove required fields, but we _might_ get away with it here due to there being so few recipe atoms in the wild. But even then the field should be commented so that someone coming along later would add a new data value at e.g. `5:` in this case.